Documentation/faq.md

```diff
@@ -1,5 +1,11 @@
 ## Frequently Asked Questions (FAQ)
 
+### etcd, general
+
+#### Does the clients need to send requests to the leader?
+
+[Raft][raft] is leader-based, and the leader handles all client requests. Client does not need to know which node is the leader. Request to follower is automatically forwarded to leader.
```

Comment:
`[Raft][raft] is leader-based; the leader handles all client requests which need cluster consensus. However, the client does not need to know which node is the leader. Any request that requires consensus sent to a follower is automatically forwarded to the leader. Requests that do not require consensus (e.g., serialized reads) can be processed by any cluster member.`

Want to clarify that not all requests go through raft...